log_stdout.module

```diff
@@ -1,23 +1,42 @@
 <?php
 
+function log_level() {
+  $severity = getenv('PENGUIN_LOG_LEVEL');
+
+  if ($severity) {
+    return defined($severity) ? constant($severity) : false;
+  }
+
+  return false;
+}
+
 function log_stdout_watchdog(array $log_entry) {
-  if ($log_entry['severity'] < WATCHDOG_WARNING) {
-    $output = fopen('php://stderr', 'w');
+  $log_level = log_level();
+
+  if (($log_level !== false) && ($log_level < $log_entry['severity'])) {
+    return;
   } else {
-    $output = fopen('php://stdout', 'w');
+    if ($log_entry['severity'] < WATCHDOG_WARNING) {
+      $output = fopen('php://stderr', 'w');
+    } else {
+      $output = fopen('php://stdout', 'w');
+    }
+
+    $severity = strtoupper(watchdog_severity_levels()[$log_entry['severity']]);
+    $user = $log_entry['user']->uid ? $log_entry['user']->name : 'anonymous';
+    $request_uri = $log_entry['request_uri'];
+    $referer_uri = $log_entry['referer'];
+    $variables = $log_entry['variables'] ? $log_entry['variables'] : array();
+
+    fwrite($output, t('WATCHDOG: [@severity] [@type] @message | user: @user | uri: @request_uri | referer: @referer_uri' . PHP_EOL, array(
```

Comment:
Vlad, unrelated to the severity level, but I noticed that the logs still don't seem to break up very well for reading, maybe we could double the PHP_EOL?